packages/amplify-cli/bin/amplify

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env node
-require = require('esm')(module, { cache: false });
+if (process.pkg) {
```

Comment:
nit: It would be better to have it inside cli's index.ts and not "externally" hooked here in a shell stub, if you can try out and it works there please move it.

Reply:
tried it out as the first line in the run() function and it didn't work so I'm going to leave it as-is